src/api/OrdersApi.js

```diff
@@ -104,14 +104,10 @@ export default class OrdersApi {
   placeOrderWithHttpInfo(id, opts) {
     opts = opts || {};
 
-    const _placeOrderRequest = PlaceOrderRequest.constructFromObject(
-      opts,
-      new CreateOrderRequest()
-    );
-    let postBody = _placeOrderRequest;
+    let postBody = opts['placeOrderRequest'];
 
     // verify the required parameter 'id' is set
-    if (id === undefined || id === null) {
+    if (_id === undefined || _id === null) {
```

Comment:
@holtbp it's very interesting that you're getting this change to. This one is wrong and breaks the functionality. The syntax is not even correct since `_id` is not defined.